src/sentry/grouping/strategies/message.py

```diff
@@ -100,7 +100,8 @@
     (?P<quoted_str>
         # The `=` here guarantees we'll only match the value half of key-value pairs,
         # rather than all quoted strings
-        ='([\w\s]+)'
+        ='([^']+)' |
+        ="([^"]+)"
```

Comment:
Cool. I see it!
<img width="1010" alt="image" src="https://github.com/getsentry/sentry/assets/44410/45805b5c-3430-4c52-a062-db825064b988">